shell/platform/darwin/ios/framework/Source/SemanticsObjectTest.mm

```diff
@@ -38,4 +38,30 @@ - (void)testCreate {
   XCTAssertNotNil(object);
 }
 
+- (void)testSetChildren {
+  fml::WeakPtrFactory<flutter::AccessibilityBridgeIos> factory(
+      new flutter::MockAccessibilityBridge());
+  fml::WeakPtr<flutter::AccessibilityBridgeIos> bridge = factory.GetWeakPtr();
+  SemanticsObject* parent = [[SemanticsObject alloc] initWithBridge:bridge uid:0];
+  SemanticsObject* child = [[SemanticsObject alloc] initWithBridge:bridge uid:1];
+  parent.children = @[child];
+  XCTAssertEqual(parent, child.parent, @"");
```

Comment:
You can leave off the @"" for all the XCAsserts, the message is optional.

Reply:
done